src/addons/presenter.js

```diff
@@ -89,7 +92,8 @@ inherit(Presenter, PureComponent, {
     return (
       createElement(PresenterMediator, {
         presenter   : this,
-        parentState : this.state
+        parentState : this.state,
+        parentProps : this.props
```

Comment:
They get passed down to bust the behavior of `React.PureComponent`. I do not think they are actually used by the mediator.